src/osm_configurator/view/toplevelframes/data_frame.py

```diff
@@ -1,15 +1,38 @@
 from __future__ import annotations
```

Comment:
alright as I see, this one is under dvelopment, so i dont reviewe this file further until it is implemented